src/domain/expense/service.py

```
@@ -117,16 +120,18 @@ async def get_by_id(self, object_id: Any, user_id: UUID) -> Expense:
+        limit: int = 20,
         date_filter: date | None = None,
         start_date: date | None = None,
         end_date: date | None = None,
         min_value: Decimal | None = None,
         max_value: Decimal | None = None,
-    ) -> list[Expense]:
-        statement = select(Expense).where(Expense.user_id == user_id)
-
+    ) -> PaginatedResponseSchema:
+        if limit < 1:
+            raise 
```

Ruff (0.15.9)

[error] Bare `raise` statement is not inside an exception handler

(PLE0704)